### ISSUE TITLE
Typo on About page?

### ISSUE THREAD

**Issue by reppep** [AsahiLinux/docs#116]
About - Asahi Linux
https://asahilinux.org/about/

I couldn't find the About page in GitHub, sorry. But Dougall isn't an architecture.

Dougall Johnson (dougallj), instruction set architecture extraordinaire.
Dougall Johnson (dougallj), instruction set architect extraordinaire.

**Comment by morsik** [comment]
But what's the issue here? Yes, Dougall is not an architecture, it's the name of the guy who worked on Apple GPU architecture so everything is correct here.

**Comment by reppep** [comment]
Now:	Dougall Johnson (dougallj), instruction set architecture extraordinaire.
Change:	Dougall Johnson (dougallj), instruction set architect extraordinaire.

	Dougall is not an architecture.

**Comment by DavidBuchanan314** [comment]
> Dougall is not an architecture.

No, but he is an ((instruction set architecture) extraordinaire).

He is not architecting anything. The ISA already exists, his role is in understanding it, etc.

"extraordinaire" is more or less equivalent to "expert" in this context. I have no idea if that's a pedantically grammatically correct usage of the phrase or not, but either way, the meaning is clear to me.

**Comment by reppep** [comment]
Architects don’t only sketch out buildings. They also inspect and propose repairs.

	If you want to keep “instruction set architecture” as a block, you need another noun for Dougall. You could call him an ISA architect extraordinaire. Or an ISA analyst would work…

Regards,

Chris
